pegasus/test/test_hoc_routes.rb

```diff
@@ -105,6 +105,33 @@ class HocRoutesTest < Minitest::Test
       end
     end
 
+    it 'starts and ends given tutorial, tracking company and tutorial' do
```

Comment:
Probably your test is being run instead of the other one?

On Wed, Dec 9, 2015 at 3:41 PM, David Bailey <notifications@github.com>
wrote:

> In pegasus/test/test_hoc_routes.rb
> <https://github.com/code-dot-org/code-dot-org/pull/6013#discussion_r47170581>
> :
>
> > @@ -105,6 +105,33 @@ class HocRoutesTest < Minitest::Test
> >        end
> >      end
> >
> > +    it 'starts and ends given tutorial, tracking company and tutorial' do
>
> as long as the test is getting run, it's good enough for now. we should
> rename one of the tests next week though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6013/files#r47170581>.
>